library/dfhack-run.cpp

```diff
@@ -43,6 +43,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <fstream>
 #include <istream>
 #include <string>
+#include <set>
```

Comment:
Just curious, did you need this to get it to compile on your end?

Reply:
Yest it does. But I am on:
gcc version 10.2.1 20200723 (Red Hat 10.2.1-1) (GCC) 

I see that you test compilation on 4.8 and it fails. That may be the issue ?

Reply:
Our CI currently uses both GCC 4.8 and 7 (but on Ubuntu). I know some Fedora fixes have been made already for GCC 10, like 7bdf958518d2892ee89a7173224a069c4a2190d8, so I was surprised that this wasn't done already. Thanks for the fix, anyway!

Reply:
Oh, I found the error on Buildmaster: https://buildmaster.lubar.me/executions/execution-details?executionId=507413&noAutoRedirect=y#S87.6619
```
Sphinx parallel build error:
RuntimeError: maximum recursion depth exceeded while pickling an object
ninja: build stopped: subcommand failed.
ninja exited with code 1 (failure)
```

This is a known (and intermittent) issue when building the documentation in the GCC 4.8 image, which is also currently stuck on an older version of Python and Sphinx. From https://buildmaster.lubar.me/applications/17/builds/build?buildId=8445, the 64-bit GCC 4.8 build succeeded (the error only happened in the 32-bit build), so I'm not worried about it.